### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by LarsJK**
Removed some redundant code and avoided some force unwrapping.

### CODE REVIEW

File: Sources/main.swift [hunk 1]
```diff
@@ -18,18 +18,18 @@ func showUsage() {
 	)
 }
 
-let arguments = Array(CommandLine.arguments.dropFirst(1))
+let arguments = CommandLine.arguments.dropFirst(1)
 
-switch arguments.stringAt(0) {
-case "--help":
+switch arguments.first {
+case "--help"?:
```
Comment (sindresorhus) [inline Sources/main.swift]: I initially had it like this, but I foolishly used `switch arguments[0]` which didn't work. Thanks for showing me the correct way.

File: Sources/DarkMode.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@ struct DarkMode {
 	}
 
 	static func toggle(force: Bool? = nil) {
-		let val = force != nil ? String(force!) : "not dark mode"
+		let val = force.map(String.init) ?? "not dark mode"
```
Comment (sindresorhus) [inline Sources/DarkMode.swift]: Nice trick!

**Comment by sindresorhus** [comment]
Thank you for the review! :) I really appreciate it. I'm just starting out with Swift, so I'm very keen on learning best practices and what to avoid.

---

I could also use a review of the Swift part of https://github.com/wulkano/aperture/pull/26 if you have any interest. No worries if not though.

**Comment by LarsJK** [comment]
Cool 👍 Will have a look at wulkano/aperture#26 later!

**Pull request merged**